src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx

```diff
@@ -27,6 +28,7 @@ const LanguageList = ({ items }) => {
         <ul className="ecl-language-list__list">
           {column2.map(item => (
             <li
+              key={item.id}
```

Comment:
the `item.label` could have been the key too ;)

Reply:
indeed...